### DISCUSSION TITLE
Can I save a har file using Taiko

### DISCUSSION THREAD

**Discussion by einkinder** [getgauge/taiko#1897] [General]
Can I save har files using Taiko and if so are there examples?

**Comment by sriv** [comment]
In theory, this should be possible using a combination of [`intercept`](https://docs.taiko.dev/api/intercept/) and an external library such as [chrome-har](https://github.com/sitespeedio/chrome-har).

If intercept does not work, you can fallback to using [`client`](https://docs.taiko.dev/api/client/).

There aren't any examples at the moment, but a PR would be most welcome. The current examples lie here: https://github.com/getgauge/taiko/tree/master/examples
